hyperstart.go

```diff
@@ -193,17 +178,18 @@ func (h *hyper) bindUnmountAllRootfs() {
 }
 
 // init is the agent initialization implementation for hyperstart.
-func (h *hyper) init(pod Pod, config interface{}) error {
+func (h *hyper) init(pod *Pod, config interface{}) error {
```

Comment:
Yes because I realized that SockCtlName and SockTtyName are empty when I need those values from register() function from proxy interface. Indeed, if those are empty, the validate() function in init() will fill them, but then, because the pod is not a pointer, we have no way to keep them...